### PR TITLE
Clean up adversarial_debiasing() function

### PR DESCRIPTION
**Pull request by josue-rodriguez**
This PR mainly addresses an issue where not all of the arguments in `adversarial_debiasing()` were being passed onto its Python counterpart

- Pass all arguments to `adversarial_debiasing()` onto `in_algo$AdversarialDebiasing()`
- Add type checks to above arguments
- Clean up function documentation
- Tidy up code to be consistent with [R coding style ](http://adv-r.had.co.nz/Style.html)

### CODE REVIEW

**Comment by SSaishruthi** [comment]
@josue-rodriguez - Can you remove the .Rhistory file?

File: aif360/aif360-r/R/inprocessing_adversarial_debiasing.R [hunk 1]
```diff
@@ -1,40 +1,73 @@
...
-adversarial_debiasing <- function(unprivileged_groups, privileged_groups, scope_name='current', sess=tf$compat$v1$Session(),
-                                  seed=NULL, adversary_loss_weight=0.1, num_epochs=50, batch_size=128,
-                                  classifier_num_hidden_units=200, debias=TRUE) {
+adversarial_debiasing <- function(unprivileged_groups, 
+                                  privileged_groups, 
+                                  scope_name = "current", 
+                                  sess = tf$compat$v1$Session(),
+                                  seed = NULL, 
+                                  adversary_loss_weight = 0.1, 
+                                  num_epochs = 50L,
```
Comment (SSaishruthi) [inline aif360/aif360-r/R/inprocessing_adversarial_debiasing.R:40]: I think it should be 50 and 128. Can you refer here?
https://github.com/Trusted-AI/AIF360/blob/bdc53541e9b4cfcfd6d1bf7923e2f147c66a1085/aif360/algorithms/inprocessing/adversarial_debiasing.py#L34

**Review by SSaishruthi — CHANGES_REQUESTED** [review]

File: aif360/aif360-r/R/inprocessing_adversarial_debiasing.R (hunk 1 above)
Comment (josue-rodriguez) [reply aif360/aif360-r/R/inprocessing_adversarial_debiasing.R:40]: Yes, in Python `50` will get treated as an integer, but in R, `50` gets treated as a double. This raises an error when running the `adversarial_debiasing()`. Setting them as e.g., `50L` ensures the arguments are treated as integers and avoids raising a Python error.

**Review by milevavantuyl — APPROVED** [review]

File: aif360/aif360-r/R/inprocessing_adversarial_debiasing.R (hunk 1 above)
Comment (SSaishruthi) [reply aif360/aif360-r/R/inprocessing_adversarial_debiasing.R:40]: Tested and works fine

**Review by SSaishruthi — APPROVED** [review]
LGTM

**Pull request merged**